### PR TITLE
misc: reduce unnecesarry output in cli/js tests

### PR DESCRIPTION
**Pull request by keroxp**
- Continuing #4148 
- `filesCopyToStdout` and stdout from `test_worker.ts` are still remaining.

### CODE REVIEW

**Comment by bartlomieju** [comment]
Great! 

> filesCopyToStdout and stdout from test_worker.ts are still remaining.

Can you address that in this PR?

**Comment by keroxp** [comment]
@bartlomieju Yes but I don't know is OK to disable. `filesCopyToStdout` is checking data flush to stdout from file itself. Output from `test_worker.ts/js` seems to be removed if they are just for debug. I think former can be fixed by changing destination to Buffer if possible.

**Comment by bartlomieju** [comment]
@keroxp just a heads up, I changed workers tests in #4209

File: cli/js/fs_events_test.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ async function getTwoEvents(
 ): Promise<Deno.FsEvent[]> {
   const events = [];
   for await (const event of iter) {
-    console.log(">>>> event", event);
+    // console.log(">>>> event", event);
```
Comment (bartlomieju) [inline cli/js/fs_events_test.ts]: Remove

File: cli/js/fs_events_test.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ testPerm({ read: true, write: true }, async function fsEventsBasic(): Promise<
 
   // We should have gotten two fs events.
   const events = await eventsPromise;
-  console.log("events", events);
+  // console.log("events", events);
```
Comment (bartlomieju) [inline cli/js/fs_events_test.ts]: Remove

File: cli/js/timers_test.ts [hunk 3]
```diff
@@ -297,7 +297,7 @@ test(async function timerMaxCpuBug(): Promise<void> {
   const { opsDispatched } = Deno.metrics();
   await waitForMs(100);
   const opsDispatched_ = Deno.metrics().opsDispatched;
-  console.log("opsDispatched", opsDispatched, "opsDispatched_", opsDispatched_);
+  // console.log("opsDispatched", opsDispatched, "opsDispatched_", opsDispatched_);
```
Comment (bartlomieju) [inline cli/js/timers_test.ts]: Remove

**Pull request merged**